block-storage/src/main/scala/coop/rchain/blockstorage/BlockStore.scala

```diff
@@ -31,6 +32,8 @@ trait BlockStore[F[_]] {
 
   def close(): F[Unit]
 
+  def iterateStream(): F[Stream[F, BlockMessage]]
```

Comment:
What we need for migration is a function like this which will be used in NodeRuntime
```scala
def migrateBlockStore(FileBS, KVBS) = ???
```
This means it will operate on implementations so there is no need to touch the BlockStore interface.

Reply:
I don't know how to migrate without offering this? Your function is too wide.
Sorry. I didn't include the function here actually.